src/components/molecules/MessageFormatting/MessageFormatting.tsx

```diff
@@ -0,0 +1,214 @@
+              ({ id, item }) => (
+                <span className={styles.inlineText} key={id}>
+                  {item}
+                </span>
+              )
+            )}
+          </code>
+        </Details.Text>
+      </Details>
+      <Details data-testid='signatures-details'>
+        <Details.Summary data-testid='signatures-summary'>
+          {messageFormattingContent.signatures.title}
+        </Details.Summary>
+        <Details.Text data-testid='signatures-text'>
+          <p>{messageFormattingContent.signatures.text}</p>
+          <code>{messageFormattingContent.signatures.codeBlockText}</code>
+        </Details.Text>
+      </Details>
+      {template === TemplateFormatText.EMAIL ? (
+        <Details data-testid='horizontal-lines-details'>
+          <Details.Summary data-testid='horizontal-lines-summary'>
+            {messageFormattingContent.horizontalLine.title}
+          </Details.Summary>
+          <Details.Text data-testid='horizontal-lines-text'>
+            <p>{messageFormattingContent.horizontalLine.text}</p>
+            <code className={styles.horizontalLine}>
+              {messageFormattingContent.horizontalLine.codeBlockText.map(
+                ({ id, item }) => (
+                  <p key={id}>{item}</p>
+                )
+              )}
+            </code>
+          </Details.Text>
+        </Details>
+      ) : null}
+    </>
+  );
+};
+const LetterFormatting = (template: string) => {
+  return (
+    <>
+      {EmailFormatting(template)}
+      <Details data-testid='page-breaks-details'>
+        <Details.Summary data-testid='page-breaks-summary'>
+          {messageFormattingContent.pageBreaks.title}
+        </Details.Summary>
+        <Details.Text data-testid='page-breaks-text'>
+          <p>{messageFormattingContent.pageBreaks.text}</p>
+          <code>
+            {messageFormattingContent.pageBreaks.codeBlockText.map(
+              ({ id, item }) => (
+                <span className={styles.inlineText} key={id}>
+                  {item}
+                </span>
+              )
+            )}
+          </code>
+        </Details.Text>
+      </Details>
+      <Details data-testid='content-blocks-details'>
+        <Details.Summary data-testid='content-blocks-summary'>
+          {messageFormattingContent.contentBlocks.title}
+        </Details.Summary>
+        <Details.Text data-testid='content-blocks-text'>
+          <p>{messageFormattingContent.contentBlocks.text1}</p>
+          <p>{messageFormattingContent.contentBlocks.text2}</p>
+          <p>{messageFormattingContent.contentBlocks.text3}</p>
+          <code>
+            {messageFormattingContent.contentBlocks.codeBlockText.map(
+              ({ id, item }) => (
+                <span className={styles.inlineText} key={id}>
+                  {item}
+                </span>
+              )
+            )}
+          </code>
+        </Details.Text>
+      </Details>
+    </>
+  );
+};
```

Comment:
i think the way this is laid out is a little confusing - eg having a component called EmailFormatting that has to check whether your template is an email because it's also used on the letter page. 

wouldn't it be clearer and simpler to just have one component with all the entries that filters what is shown based on your template type?

Reply:
Refactored